src/olympia/addons/templates/addons/replacement_addons.html

```diff
@@ -0,0 +1 @@
+{% extends "impala/base_shared.html" %}
```

Comment:
Is this file extended instead of `impala/base.html` because base.html is ambiguous?